pkg/test/expectations/expectations.go

```diff
@@ -568,3 +568,14 @@ func ExpectMakeNodesAndNodeClaimsInitializedAndStateUpdated(ctx context.Context,
 		ExpectReconcileSucceeded(ctx, nodeClaimStateController, client.ObjectKeyFromObject(m))
 	}
 }
+
+func ExpectEvicted(ctx context.Context, c client.Client, pods ...*v1.Pod) {
+	GinkgoHelper()
+	for _, pod := range pods {
+		Eventually(func() bool {
+			return ExpectPodExists(ctx, c, pod.Name, pod.Namespace).GetDeletionTimestamp().IsZero()
+		}, ReconcilerPropagationTime, RequestInterval).Should(BeFalse(), func() string {
+			return fmt.Sprintf("expected %s/%s to be evicting, but it isn't", pod.Namespace, pod.Name)
```

Comment:
nit: I was originally confused because I saw that we were expecting it to exist and not have deletion timestamp, but saw that we were expecting it to be false. I think the converse would semantically make more sense, but it's up to you

Reply:
Agreed. Funny enough, I also got confused and thought that the logic was wrong here. This is technically not a change from the existing code, though; this is just a move to get this function over into the expectations file